### ISSUE TITLE
Always `false` + different props than docs

### ISSUE THREAD

**Issue by marcelscruz** [lsirivong/gatsby-plugin-modal-routing#59]
For some reason, I don't seem to be able to get it to work.

I'm currently using the same package version as the example project (`1.1.0-beta`), even though I tried other versions too, and even copying the exact code from the example, but it doesn't work.

In fact, the props I receive from the `ModalRoutingContext.Consumer` is different from the one detailed in the docs.

<img width="398" alt="Screenshot 2021-03-11 at 15 43 29" src="https://user-images.githubusercontent.com/12413903/110838075-d079cc80-8280-11eb-89d0-41357077bbec.png">

I receive `isModal` instead of `modal`.

And `isModal` is always `false`.

Am I missing something here?

Any help is very appreciated. Thanks!

**Comment by coreAux** [comment]
Same here. Resolved it by following the steps in [this thread](https://github.com/lsirivong/gatsby-plugin-modal-routing/issues/57).

TL;DR: If you're running Gatsby v.3 make sure to install `gatsby-plugin-modal-routing-3` and replace all imports/configs.

**Comment by andreisemianiuk** [comment]
I have the same problem! How to fix it?

**Comment by coreAux** [comment]
> I have the same problem! How to fix it?

@andreisemianiuk what version Gatsby are you using and what version plugin?
Have you tried following the step in [this thread](https://github.com/lsirivong/gatsby-plugin-modal-routing/issues/57#issuecomment-840020305)?

**Comment by andreisemianiuk** [comment]
> 
"gatsby": "^4.9.3",
gatsby-plugin-modal-routing-3
I've tried [this thread](https://github.com/lsirivong/gatsby-plugin-modal-routing/issues/57#issuecomment-840020305), but had the same problem. Actually I don't use it now

**Comment by otkstudio** [comment]
@marcelscruz Where you able to resolve this? The steps in the mentioned thread didn't solve my problem. I have the exact issue you had.

**Comment by marcelscruz** [comment]
@otkstdio not really, never found a fix and then moved to a different implementation.

**Comment by otkstudio** [comment]
@marcelscruz Would you mind sharing the implementation you went with? I'm getting tired of attempting to make this package work. It's become very frustrating.

**Comment by marcelscruz** [comment]
@otkstdio sorry, I meant I gave up what I was trying to do 😅
